packages/plus/src/components/CodeEditor/CodeEditor.tsx

```diff
@@ -34,27 +119,57 @@ export const CodeEditor = ({
   height,
   readOnly,
   autoCompletion,
+  disabled = false,
```

Comment:
There is still an issue with disabled mode I can select the content, this should be in readonly instead

![Screenshot 2025-05-05 at 09 17 44](https://github.com/user-attachments/assets/6e90ad1c-b5f3-485a-81bf-eead89f02390)